docs-2.0/20.appendix/6.eco-tool-version.md

```diff
@@ -50,6 +50,8 @@ Nebula Dashboard（简称 Dashboard）是一款用于监控 Nebula Graph 集群
 |:---|:---|
 | {{ nebula.tag }}  | {{dashboard.tag}}|
 
+{{ ent.ent-begin }}
+
 ## Nebula Dashboard（企业版）
```

Comment:
is there any problem if we add macro for dashboard enterprise edition and explorer, since we release these two products.